lib/modules/datasource/galaxy-collection/index.ts

```diff
@@ -2,22 +2,29 @@ import is from '@sindresorhus/is';
 import { logger } from '../../../logger';
 import { cache } from '../../../util/cache/package/decorator';
 import * as p from '../../../util/promises';
+import { regEx } from '../../../util/regex';
 import { ensureTrailingSlash, joinUrlParts } from '../../../util/url';
 import * as pep440Versioning from '../../versioning/pep440';
 import { Datasource } from '../datasource';
 import type { GetReleasesConfig, Release, ReleaseResult } from '../types';
 import { GalaxyV3, GalaxyV3DetailedVersion, GalaxyV3Versions } from './schema';
 
+const repositoryRegex = regEx(
+  /^[\S]+\/api\/galaxy\/content\/(?<repository>[^/]+)/,
+);
+
 export class GalaxyCollectionDatasource extends Datasource {
   static readonly id = 'galaxy-collection';
 
   constructor() {
     super(GalaxyCollectionDatasource.id);
   }
 
-  override readonly customRegistrySupport = false;
+  override readonly customRegistrySupport = true;
+
+  override readonly registryStrategy = 'hunt';
```

Comment:
Why do you want to use `hunt`?

Reply:
Using `first` doesn't work because renovate should check all available registryUrls for a collection. `hunt` would be the correct strategy for that, right?